src/App.tsx

```diff
@@ -1,14 +1,85 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+
+import { Todo } from './types/Todo';
+import { SortType } from './types/SortType';
 
 export const App: React.FC = () => {
+  const [loading, setLoading] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```
![image](https://user-images.githubusercontent.com/96409563/211373826-495f12aa-b284-4713-9207-5bf3003d1e1a.png)